src/transformers/modeling_layoutlm.py

```diff
@@ -94,10 +94,15 @@ def forward(
 
         words_embeddings = inputs_embeds
         position_embeddings = self.position_embeddings(position_ids)
-        left_position_embeddings = self.x_position_embeddings(bbox[:, :, 0])
-        upper_position_embeddings = self.y_position_embeddings(bbox[:, :, 1])
-        right_position_embeddings = self.x_position_embeddings(bbox[:, :, 2])
-        lower_position_embeddings = self.y_position_embeddings(bbox[:, :, 3])
+        try:
+            left_position_embeddings = self.x_position_embeddings(bbox[:, :, 0])
+            upper_position_embeddings = self.y_position_embeddings(bbox[:, :, 1])
+            right_position_embeddings = self.x_position_embeddings(bbox[:, :, 2])
+            lower_position_embeddings = self.y_position_embeddings(bbox[:, :, 3])
+        except IndexError as e:
+            print("The :obj:`bbox`coordinate values should be within 0-1000 range.")
+            raise e
```

Comment:
Better to chain errors than to print imo:

```suggestion
        except IndexError as e:
            raise IndexError("The :obj:`bbox`coordinate values should be within 0-1000 range.") from e
```

Reply:
Thank you!